source/connect.txt

```diff
@@ -105,6 +105,46 @@ deployment from which you were disconnected:
 
 .. include:: /includes/steps/activate-connection.rst
 
+Launch MongoDB Shell
+~~~~~~~~~~~~~~~~~~~~
+
+You can connect the
+`MongoDB Shell <https://docs.mongodb.com/mongodb-shell/>`__ or legacy
+:binary:`mongo <mongo>` shell to your current active deployment.
+
+Considerations
+``````````````
+
+- |vsce| determines which shell to use to connect to your deployment
+  by the :guilabel:`Shell` setting in your
+  :ref:`extension settings <vsce-settings>`. You can choose either the
+  `MongoDB Shell <https://docs.mongodb.com/mongodb-shell/>`__ or legacy
+  :binary:`mongo <mongo>` shell.
+
+- To successfully utilize this feature, the path to your desired shell
+  must exist in your system's ``PATH``. If the shell specified in your
+  :ref:`settings <vsce-settings>` does not exist in your ``PATH``,
+  |vsce| attempts to use the other shell. If neither shell exists in
+  your ``PATH``, the operation errors.
+
+Procedure
+`````````
+
+To launch the MongoDB Shell connected to your active deployment:
```

Comment:
[opinion] Here and below it reads just a little strangely to me to say the shell launches in a connected state - I think it's perfectly clear so just calling out in case you feel the same.

I'd say "To launch the MongoDB Shell and connect to..."

Reply:
My hesitation with this suggestion is it kind of overlaps with connecting in VSCE. They've already connected via VSCE, so I think this wording might confuse users who skip right to the procedure and think they can connect to any deployment regardless of it is active or not. Going to leave this one as-is.

Reply:
Actually, I think we can just say "To connect the shell to your active deployment..." trims some words and I think is clear.